llkv-tpch/DRAFT-QUERY-LEVEL-CONTEXT.md

```diff
@@ -0,0 +1,50 @@
+# Per-Query Perf Outline
```

Comment:
This was intentionally added, as I will likely document per-query optimization ideas.